src/upgrade/cve.ts

```diff
@@ -0,0 +1,154 @@
+import { UpgradeIssue, UpgradeReason } from "./type";
+import { Octokit } from "@octokit/rest";
+import * as semver from 'semver';
+
+/**
+ * Severity levels ordered by criticality (higher number = more critical)
+ * The official doc about the severity levels can be found at:
+ * https://docs.github.com/en/rest/security-advisories/global-advisories?apiVersion=2022-11-28
+ */
+export const severityOrder = {
+    critical: 4,
+    high: 3,
+    medium: 2,
+    low: 1,
+    unknown: 0,
+} as const;
+
+export type Severity = keyof typeof severityOrder;
```

Comment:
may be simplified using numeric enum
https://www.typescriptlang.org/docs/handbook/enums.html#numeric-enums